.gitlab/helper_functions.sh

```diff
@@ -62,5 +62,5 @@ section_end() {
 
 job-done() {
     mkdir -p "${PIPELINE_JOBS_FOLDER}"
-    echo "done" > ${PIPELINE_JOBS_FOLDER}/${CI_JOB_NAME}.txt
+    echo "done" > "${PIPELINE_JOBS_FOLDER}/${CI_JOB_NAME}.txt"
```

Comment:
why with quotes?

Reply:
@esharf added this, it's because of a failure we got "ambiguous redirect"